Dockerfile

```diff
@@ -55,9 +55,8 @@ RUN apk update && \
     apk add --no-cache bash ca-certificates curl procps unzip wget && rm -rf /var/cache/apk/* \
     && wget -qO- https://aka.ms/getvsdbgsh | /bin/sh /dev/stdin -v latest -l /vsdbg \
     && wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub \
-    && wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.29-r0/glibc-2.29-r0.apk \
-    # see https://github.com/sgerrand/alpine-pkg-glibc/issues/185 for need for force-overwrite
-    && apk add --force-overwrite glibc-2.29-r0.apk
+    && wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.35-r1/glibc-2.35-r1.apk \
+    && apk add --no-cache glibc-2.35-r1.apk
```

Comment:
This issue is now resolved for our use case in the latest version so no need to force overwrite anymore.